easybuild/tools/toolchain/toolchain.py

```diff
@@ -796,6 +797,12 @@ def prepare_rpath_wrappers(self, rpath_filter_dirs=None):
         rpath_filter = ','.join(rpath_filter + ['%s.*' % d for d in rpath_filter_dirs or []])
         self.log.debug("Combined RPATH filter: '%s'", rpath_filter)
 
+        # figure out list of patterns to use in rpath include
+        # rpath_include = build_option('rpath_include')
+        rpath_include = ','.join(['%s' % d for d in rpath_include_dirs or []])
```

Comment:
`'%s' % d` doesn't do much, since the directories are already strings, no?

So this is equivalent with:

```python
rpath_include = ','.join(rpath_include_dirs or [])
```